File: Python/card.py
from payment import Payment

class Card(Payment):
    ident = int
    cardName = str
    cardNumber = int
    cvv = int
    expirationDate = int

    def __init__(self, id, cardName, cardNumber, cvv, expirationDate):
        super().__innit__(ident)
        self.cardName = cardName
        self.cardNumber = cardNumber
        self.cvv = cvv
        self.expirationDate = expirationDate

File: Java/Card.java
class Card extends Payment {
    Integer id;
    String cardName;
    Integer cardNumber;
    Integer cvv;
    Integer expirationDate;

    public Card(Integer id, String cardName, Integer cardNumber, Integer cvv, Integer expirationDate){
        super(id);
        this.cardName = cardName;
        this.cardNumber = cardNumber;
        this.cvv = cvv;
        this.expirationDate = expirationDate;
    }
}
